lib/phoenix_profiler/telemetry.ex

```diff
@@ -0,0 +1,90 @@
+defmodule PhoenixProfiler.Telemetry do
+  # Telemetry helpers
+  @moduledoc false
+
+  live_view_events =
+    for stage <- [:mount, :handle_params, :handle_event],
+        action <- [:start, :stop, :exception] do
+      [:phoenix, :live_view, stage, action]
+    end
+
+  plug_events = [
+    [:phoenix, :endpoint, :stop],
+    [:phxprof, :plug, :stop]
+  ]
+
+  @events plug_events ++ live_view_events
+
+  @doc """
+  Returns a list of built-in telemetry events to collect.
+  """
+  def events, do: @events
+
+  @doc """
+  Collector filter callback.
+  """
+  def collect(_, [:phoenix, :endpoint, :stop], %{duration: duration}, _meta) do
+    {:keep, %{endpoint_duration: duration}}
+  end
+
+  def collect(_, [:phxprof, :plug, :stop], measures, %{conn: conn}) do
+    profile = conn.private.phoenix_profiler
+
+    case profile.info do
+      :disable ->
+        :skip
+
+      info when info in [nil, :enable] ->
+        {:keep,
+         %{
+           at: profile.system_time,
+           conn: %{conn | resp_body: nil, assigns: Map.delete(conn.assigns, :content)},
+           metrics: %{
+             memory: collect_memory(conn.owner),
```

Comment:
I see no `collect_memory` for live view events, so that means `memory` is collected on the first page load only right? What about we start updating memory on each LV event too? Certainly some LV actions would make use of that information :)

Reply:
Sounds good 👍🏻